social/tasks.py

```diff
@@ -0,0 +1,44 @@
+import feedparser
+import re
+
+from background_task import background
+
+from social.app.models.author import Author
+from social.app.models.post import Post
+
+# Get the GitHub activity of a user
+# Reference source: https://pypi.python.org/pypi/django-background-tasks
+@background(schedule=60)
```

Comment:
`schedule=60` means it'll just get called once 60 seconds after it gets scheduled. We're going to need a repeating schedule, no?